src/test/java/com/amihaiemil/zold/mock/MockHttpClient.java

```diff
@@ -74,72 +74,67 @@ public MockHttpClient(final AssertRequest... assertions) {
     }
 
     @Override
-    public HttpParams getParams() {
```

Comment:
@wholegroup , could you please clarify whether the removal of the `getParams` and `getConnectionManager` was intentional and if so, provide an explanation on that?

Reply:
API was changed and there is no such method in the HttpClient interface anymore.